README.md

```diff
@@ -59,6 +59,21 @@ Go through this checklist after creating your repository. It should only take a
 - [ ] Have you added tests? Are they matched, here?
 - [ ] Have you added a `lint` command, if using [`eslint-config-orbitdb`](https://github.com/orbitdb/eslint-config-orbitdb)?
 
+### Tests
+
+- [ ] Are there automated tests?
+- [ ] ...for the browser as well?
+- [ ] Are those reflected in CI?
+- [ ] Bonus points: Using CircleCI workflows to segment tests?
+- [ ] Extra bonus points: Are you cross-testing dependencies (i.e. are changes in `orbit-db-keystore` tested in `orbit-db` as well
+
+### Benchmarks
+- [ ] Are there benchmarks?
+
```

Comment:
Proposal to add: "[ ] Did you run the benchmarks before and after the change/PR"

Reply:
added!